src/content/app-platform/create-catalog/index.md

```diff
@@ -0,0 +1,128 @@
+---
+linkTitle: Creating an app catalog
+title: Creating an app catalog
+description: How to create a custom app catalog for use with app platform and push helm charts to it.
+weight: 40
+menu:
+  main:
+    parent: app-platform
+last_review_date: 2021-12-01
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+user_questions:
+  - How can I create an app catalog for my apps?
+  - How can I serve an app catalog using GitHub Pages?
+  - How can I create a catalog CR for my app catalog?
+  - How can I push an app to my app catalog?
+---
+
+# Creating an app catalog
+
+## Overview
+
+An app catalog is a collection of apps that can be deployed using the Giant Swarm App Platform.
+We fully support Helm and an app catalog is also a Helm [Chart Repository](https://helm.sh/docs/topics/chart_repository/).
+Each app catalog has its own [catalog]({{< relref "/ui-api/management-api/crd/catalogs.application.giantswarm.io.md" >}})
```

Comment:
Should we make it capital 'C' - `Catalog`, like the full CR name?

Reply:
I lose track of what style we use :) I'll check 👍

Reply:
We use caps. I've updated.